frontend/src/app/survey/tabs/dashboard/notification/notification.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-notification',
+  templateUrl: './notification.component.html',
+  styleUrls: ['./notification.component.sass']
+})
+export class NotificationComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  notifications = ["Hallo, Willkommen", "Irgendjemand ist blöd"]
```

Comment:
This looks like test-data to me. Should probably be either an empty array or some external properties.